pages/api/request/help.ts

```diff
@@ -0,0 +1,88 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { cors, middleware, sessionExists } from "@lib/middleware";
+import { NotifyClient } from "notifications-node-client";
+import { logMessage } from "@lib/logger";
+import { MiddlewareProps } from "@lib/types";
+
+const SUPPORT_EMAIL_ADDRESS = "assistance+forms@cds-snc.freshdesk.com";
+const CONTACTUS_EMAIL_ADDRESS = "jose.jimenez@cds-snc.ca";
+
+const requestSupport = async (
+  req: NextApiRequest,
+  res: NextApiResponse,
+  { session }: MiddlewareProps
+) => {
+  try {
+    if (!session) return res.status(403).json({});
```

Comment:
Not sure you need this check. you do have the sessionExists middleware which does it

Reply:
You need it because the session is still an optional value when retrieved from the `MiddlewareProps`. This was the reason why I wanted to introduce the `assert(session)` at some point (we had a discussion on that topic).

Reply:
But let me see if I can solve this problem more generally. We shouldn't need a check to account for TypeScript since its not actually present in the code when it runs. We know the session will always be passed down 

I can define the following type `type WithRequired<T, K extends keyof T> = T & Required<Pick<T, K>>`  Which would allow us to use it like this 

```TS
{ session } :  WithRequired<MiddlewareProps, "session">
```

No more TS errors. I will open a PR for this

Reply:
That is nice!

Reply:
https://github.com/cds-snc/platform-forms-client/pull/1374

Reply:
Would this apply to a user that is not authenticated? My assumption was no

Reply:
Hey Peter. If a user is unauthenticated there would be no session and thus the sessionCheck would return a 403 error